references/common/application.go

```diff
@@ -73,23 +87,118 @@ type DeleteOptions struct {
 	CompName  string
 	Client    client.Client
 	C         common.Args
+
+	Wait        bool
+	ForceDelete bool
 }
 
 // DeleteApp will delete app including server side
-func (o *DeleteOptions) DeleteApp() (string, error) {
+func (o *DeleteOptions) DeleteApp(io cmdutil.IOStreams) error {
+	if o.Wait {
+		return o.WaitUntilDeleteApp(io)
+	}
+	if o.ForceDelete {
+		return o.ForceDeleteApp(io)
+	}
```

Comment:
Maybe user can use these two flags together? He may both want to wait and force delete?

Reply:
There is no waiting for forced deletion